common/options.go

```diff
@@ -14,6 +16,14 @@ const (
 	UrlParameterOnFinalBlock = "onFinalBlock"
 	// UrlParameterOnStartOfEpoch represents the name of an URL parameter
 	UrlParameterOnStartOfEpoch = "onStartOfEpoch"
+	// UrlParameterBlockNonce represents the name of an URL parameter
```

Comment:
kind of dummy comments :(

Reply:
Unfortunately, I cannot think to another commenting approach in this particular context - and also keeping the comment - code entropy to a decently-low level (without restating the actual value of the string in the comment).

Reply:
can be done in another PR